### ISSUE TITLE
[Consistency] User encounters content with errors or inconsistencies.

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#78952]
_This ticket was generated during the pilot process for the new VA.gov experience standards. Please tag `@platform-governance-team-members` on Slack if you need assistance interpreting this ticket._

## Product Information
**Team:** Identity Core Team
**Product:** Terms of Use (ToU)
**Feature:** Terms of Use (ToU)

## Findings details
**VA.gov Experience Standard - issue:** User encounters content with errors or inconsistencies.
**VA.gov Experience Standard - category:** Consistency
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @rferguson13 (Content)

### Description

Use of Contact Information - The user finds the following sentence: "We may contact you at the email address, phone numbers, or addresses you provide to us." "Email address" is the only singular noun on the list; can there ever be multiple email addresses? 

## Recommended action
For consistency, please revise to the following: "We may contact you at the email addresses…"

## References

- **WCAG Success Criteria:** N/A
- **Modality:** N/A
- **Design System Component:** N/A
- **Design System Pattern:** N/A
- **Content Style Guide:** [Content principles](https://design.va.gov/content-style-guide/content-principles)
- **Context:** N/A
- **Accessibility Defect Severity:** N/A

---

### Next Steps for the VFS Team

- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by asg5704** [comment]
[This is the reference document](https://docs.google.com/document/d/1S2MTIZ3tklLheHF--xQQau9PnVkVUlNd/edit?usp=sharing&ouid=105984809890077019983&rtpof=true&sd=true) provided to the Identity-team for the Terms of Use content. (Content was modified by @DanielleThierryUSDSVA and @laurwill in compliance with Office of General Counsel (OGC).

I can absolutely fix "the bold" once OGC & the above people sign off on it and provide us with an updated Terms of Use document. But until that time I'm going to be putting this in the "Won't Fix" category.

**Comment by rferguson13** [comment]
Hi @DanielleThierryUSDSVA and @laurwill ! I was wondering if you had a chance to look at @asg5704 's comment yet. What would good next steps be?

**Comment by laurwill** [comment]
Hi, thanks for the tag! I don't think I've seen any VA forms or tools that collect multiple email addresses from Veterans, but we do collect multiple addresses and phone numbers. So I think it's accurate for that to be the only singular noun in that sentence. 

Also @asg5704 I see that you're referring to a Google doc for the legal terms source of truth. We all need to be using this [Sharepoint document](https://gcc02.safelinks.protection.outlook.com/ap/w-59584e83/?url=https%3A%2F%2Fdvagov-my.sharepoint.com%2F%3Aw%3A%2Fg%2Fpersonal%2Flaura_willwerth_va_gov%2FEXsNiSN-vA1CoKAmXPOd6mMB_t4ZGnw4fkReC9fNl5rvAw%3Fe%3Dctnbg4&data=05%7C01%7C%7C684474dd629b4efdd4fd08db62164999%7Ce95f1b23abaf45ee821db7ab251ab3bf%7C0%7C0%7C638211620079434884%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=sP4E7sF9sw4kKcUn1Bqozjz%2FzoqzaxVji%2BEF1%2F1FIUA%3D&reserved=0) as the source of truth for the legal terms. I'm not sure when you moved into the Google doc, but the Sharepoint version is the source of truth.

Danielle is also working on some final changes to the legal terms from SMEs — she'll share those as soon as they're ready, probably later today.

Let me know if you have any questions in the meantime!

**Comment by rferguson13** [comment]
Hi @laurwill - Thank you for clarifying! @asg5704 You can't put this in the "Won't Fix" category.

**Comment by asg5704** [comment]
@laurwill - That Google docs was imported from the last official document my Program Manager (Tom Black) received from Danielle Thierry, OGC, and your team. You are stating there are changes to the Terms of Use document - none of those were communicated to myself nor my team which is highly concerning.

@rferguson13 - To clarify, when I put this in the "Won't Fix" category. I mean there is a launch date of 4/4 at 9PM EST. So the changes you are requesting will NOT make it into the launch date, it will be done after launch. Hope this helps.

**Comment by laurwill** [comment]
Hi @asg5704 ,
Totally understand the frustration with the last-minute changes, but these were unexpected. When we passed you the "clean for final approval" version via Sharepoint, that was the final text to the best of our knowledge, and we let you know as soon as we found out about the edits. Unfortunately these types of late updates do happen, particularly with this type of legal text that several groups of SMEs need to approve. 

Danielle posted a new Sharepoint doc here this morning: https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/content/terms-of-use/latest-version.md This highlights the minor changes to the text. Let me know if you have any questions about the changes — I know your team is on a tight timeline, so we'll help however we can.

And just to clarify about post-launch changes — we can't make changes to the legal terms (other than any true errors/typos) without getting approval from the SMEs who have reviewed and approved the terms. If there are any other places in the text you have concerns about, happy to chat about those and help decide if any action is needed!
